lib/Client.js

```diff
@@ -179,7 +179,9 @@ class Client extends EventEmitter {
             if(Array.isArray(this.options.intents)) {
                 let bitmask = 0;
                 for(const intent of this.options.intents) {
-                    if(Constants.Intents[intent]) {
+                    if(typeof intent === "number") {
+                        bitmask |= (1 << intent);
```

Comment:
People will almost certainly assume you should provide the full number for the intent, not the number that's shifted to get the final result (e.g. `32768` instead of `15`)

In my mind that behavior makes more logical sense